lms/models/public_id.py

```diff
@@ -1,100 +0,0 @@
-from base64 import urlsafe_b64encode
-from dataclasses import dataclass
-from uuid import uuid4
-
-from lms.models.region import Region
-
-
-class InvalidPublicId(Exception):
-    """Indicate an error with the specified public id."""
-
-
-@dataclass
-class PublicId:
```

Comment:
So not only was there a `PublicIdMixin` but there's also a separate dataclass called `PublicId`? It's so hard to follow! Glad to see this gone as well